src/main/java/frc/robot/robotcontainers/MiniSwerveContainer.java

```diff
@@ -11,17 +11,15 @@
 import edu.wpi.first.wpilibj.XboxController;
 import frc.robot.Robot;
 import frc.robot.auto.AutoSelector;
-import frc.robot.commands.ModuleAngleFromJoystick;
-import frc.robot.commands.ResetGyro;
-import frc.robot.commands.SwerveTeleop;
 import frc.robot.config.Config;
 import frc.robot.subsystems.SwerveModule;
 import frc.robot.subsystems.SwerveSubsystem;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
 import edu.wpi.first.wpilibj2.command.button.JoystickButton;
-
+import frc.robot.commands.*;
```

Comment:
Depending on how many commands you need from frc.robot.commands, it's better practice to import the items you need individually, rather than using * to import the whole thing.